aiogram/webhook/aiohttp_server.py

```diff
@@ -185,6 +189,8 @@ async def _handle_request(self, bot: Bot, request: web.Request) -> web.Response:
 
     async def handle(self, request: web.Request) -> web.Response:
         bot = await self.resolve_bot(request)
+        if not self.verify_secret(request.headers.get("X-Telegram-Bot-Api-Secret-Token")):
+            return web.Response(body="Unauthorized", status=401)
```

Comment:
So what if I don't wanna verify secrets?

Reply:
i dont have ideas :(

Reply:
You should return `True` by default from `verify_secret` method

Reply:
I think we should make a single choice of implementation level.
- `self.verify_secret` implemented on base level
- `self.secret_token` implemented on upper level

So, we can push down secret token and check it's not None before trying to get headers and verify something.